Cargo.toml

```diff
@@ -39,30 +39,23 @@ incremental = false
 panic = "abort"
 
 [features]
-default = ["ring-aead-ciphers", "sodium", "rc4", "aes-cfb", "aes-ctr", "trust-dns", "local-http", "local-http-rustls", "local-tunnel", "local-socks4"]
-# Enables AEAD ciphers that depending on ring
-# aes-128-gcm, aes-256-gcm, chacha20-ietf-poly1305
-ring-aead-ciphers = ["ring"]
-# Enables ciphers that depending on libsodium
-# chacha20, xchacha20, chacha20-ietf
-# salsa20, xsalsa20
-# xchacha20-ietf-poly1305
-sodium = ["libsodium-sys"]
-rc4 = ["openssl"]
-aes-cfb = ["openssl"]
-aes-ctr = ["openssl"]
-aes-pmac-siv = ["miscreant"]
-camellia-cfb = ["openssl"]
+default = [
+    "clap/wrap_help",
+    "clap/suggestions",
+    "serde/derive",
+    "tokio/full",
```

Comment:
It shouldn't work if users compile with `--no-default-features` without `tokio/full`